kev1n1in/AsiaYoTravelMap
# Feature/modal

[Website URL]()
## Description
* Complete the modal layout 
*  Install MUI date and time pickers

## Test Plan
1. Click on any attraction pin.
2. Verify that the image, name, and address are displayed correctly.
3. Select the journey date and time.
4. Click "Create Trip" and confirm that it has been successfully added.
